charts/gardener-extension-os-coreos/templates/vpa.yaml

```diff
@@ -1,13 +1,22 @@
----
+{{- if .Values.vpa.enabled}}
 apiVersion: "autoscaling.k8s.io/v1beta2"
 kind: VerticalPodAutoscaler
 metadata:
-  name: gardener-extension-os-coreos-vpa
+  name: {{ include "name" . }}-vpa
   namespace: {{ .Release.Namespace }}
 spec:
+  {{- if .Values.vpa.resourcePolicy }}
+  resourcePolicy:
+    containerPolicies:
+      - containerName: '*'
+        minAllowed:
+          cpu: {{ required ".Values.vpa.resourcePolicy.minAllowed.cpu is required" .Values.vpa.resourcePolicy.minAllowed.cpu }}
+          memory: {{ required ".Values.vpa.resourcePolicy.minAllowed.memory is required" .Values.vpa.resourcePolicy.minAllowed.memory }}
```

Comment:
```suggestion
          cpu: {{ required "vpa.resourcePolicy.minAllowed.cpu is required" .Values.vpa.resourcePolicy.minAllowed.cpu }}
          memory: {{ required "vpa.resourcePolicy.minAllowed.memory is required" .Values.vpa.resourcePolicy.minAllowed.memory }}
```

Reply:
Is there a reason why the `.Values.` should not be used in this case? in the rest of the file (and in the other template files) there is everywhere a `.Values.`.

Reply:
No particular reason, this is just the way that it is usually done in the charts definitions in https://github.com/gardener